pkg/provider/wechat/parse.go

```diff
@@ -31,7 +32,9 @@ func (w *Wechat) translateToOrders(array []string) error {
 	bill.TxType = getTxType(array[1])
 	switch bill.TxType {
 	case TxTypeCash2Cash:
-		fmt.Printf("Get an unusable tx type, ignore it: %s\n", bill.TxType)
+		fallthrough
+	case TxTypeCash2Cash_2:
```

Comment:
Can we have a better variable name for it?

Reply:
Maybe TxTypeCash2CashLooseChange

Reply:
🤣 Agreed

Reply:
and fixed.